### ISSUE TITLE
Debugging es6 file does not pause on start

### ISSUE THREAD

**Issue by blake-regalia** [CrabDude/babel-node-debug#13]
Loading the inspector requires "pausing" before I can hit "resume", which then pauses on first line with `--debug-brk` option.

_test.js_

``` js
(function(b) {
    let a = b+'!';
    debugger;
})('test');
```

``` sh
$ bode-debug --no-preload --cli --debug-brk test.js
```

<img width="314" alt="screen shot 2016-02-05 at 12 20 30 pm" src="https://cloud.githubusercontent.com/assets/1456400/12858115/efe38fa0-cc02-11e5-8660-e4759f375845.png">

Not a critical component, but a bit redundant to have to pause/resume instead of just resume from debug start.

**Comment by CrabDude** [comment]
Yes. It's super annoying. I don't know why that is, but the whole toolchain is rather obtuse.

I'm going to try and set the breakpoint through the front-end instead of V8's Debugging Protocol, so it functions more closely to a non-source-mapped file.
